stdpopsim/catalog/HomSap/species.py

```diff
@@ -76,6 +82,7 @@
             synonyms=data["synonyms"],
             mutation_rate=1.29e-8,
             recombination_rate=_recombination_rate_data[name],
+            ploidy=_ploidy_data[name] if name in _ploidy_data.keys() else None,
```

Comment:
@grahamgower, is this what you mean? If ploidy is not the default (for a species) then set `Chromosome.ploidy` from a dict defined in `catalog/*/species.py` -- and when making a contig use `Species.ploidy` if `Chromosome.ploidy` is `None`.

It'd be nice to have chromosome ploidy automatically generated by the maintenance scripts, but I'm not sure that's available through the REST API.

Reply:
... and per-species QC like,
https://github.com/popsim-consortium/stdpopsim/blob/b63a11a2b6841fd789f8152ea22db8deceaeb536/tests/test_HomSap.py#L41-L48

Reply:
Sorry, I wasn't suggesting to get the ploidy info from ensembl. I was only thinking of doing some dumb name matching magic in the maintenance script---but what you're doing here is just fine.